src/utils/leftPad.ts

```diff
@@ -1,5 +1,5 @@
-export const leftPad = (number: number, targetLength: number) => {
-  let output = number + "";
+export const leftPad = (number: number, targetLength: number): string => {
```

Comment:
Thanks, this is way much cleaner than having to convert the numbers into string again in the place where i called it!